src/components/mdx-components/component-links.tsx

```diff
@@ -1,144 +1,150 @@
+      minH='32px'
+      borderWidth='1px'
+      borderRadius='md'
+      color={useColorModeValue('gray.600', 'whiteAlpha.700')}
+      _hover={{
+        color: useColorModeValue('gray.700', 'whiteAlpha.900'),
+        boxShadow: 'sm',
+        transform: 'translateY(-1px)',
+      }}
+      {...rest}
+    >
+      <HStack>
+        <Icon fontSize={iconSize} as={icon} color={iconColor} />
+        <Text fontSize='sm' lineHeight='short'>
+          {children}
+        </Text>
+      </HStack>
+    </Link>
+  )
+}
+
+export type ComponentLinksProps = {
+  theme?: { componentName: string }
+  github?: { url?: string; package?: string }
+  npm?: { package: string }
+  storybook?: { url: string }
+  video?: { url: string }
+}
+function ComponentLinks(props: ComponentLinksProps) {
+  const { theme, github, npm, storybook, video, ...rest } = props
+  const iconColor = useColorModeValue('gray.600', 'inherit')
+
+  const githubRepoUrl = 'https://github.com/chakra-ui/chakra-ui'
+
+  const githubLink = (github?.url || github?.package) && (
+    <WrapItem>
+      <ComponentLink
+        url={
+          github.url || `${githubRepoUrl}/tree/main/packages/${github.package}`
+        }
+        icon={FaGithub}
+        iconColor={iconColor}
+        iconSize='1rem'
+      >
+        {t('component.mdx-components.component-links.view-source')}
+      </ComponentLink>
+    </WrapItem>
+  )
+
+  const npmLink = npm?.package && (
+    <WrapItem>
+      <ComponentLink
+        url={`https://www.npmjs.com/package/${npm.package}`}
+        icon={FaNpm}
+        iconSize='2rem'
+        iconColor='red.500'
+      >
+        {npm.package}
+      </ComponentLink>
+    </WrapItem>
+  )
+
+  const storybookLink = storybook?.url && (
+    <WrapItem>
+      <ComponentLink
+        url={storybook.url}
+        icon={StorybookIcon}
+        iconSize='1.25rem'
+        iconColor='pink.500'
+      >
+        {t('component.mdx-components.component-links.view-storybook')}
+      </ComponentLink>
+    </WrapItem>
+  )
+
+  const videoLink = video?.url && (
+    <WrapItem>
+      <ComponentLink
+        url={video.url}
+        icon={FaYoutube}
+        iconSize='1.2rem'
+        iconColor='red.500'
+      >
+        {t('component.mdx-components.component-links.view-video')}
+      </ComponentLink>
+    </WrapItem>
+  )
+
+  const themeComponentLink = theme && (
+    <WrapItem>
+      <ComponentLink
+        url={`${githubRepoUrl}/tree/main/packages/theme/src/components/${theme.componentName}.ts`}
+        icon={FaGithub}
+        iconColor={iconColor}
+        iconSize='1rem'
+      >
+        {t('component.mdx-components.component-links.view-theme-source')}
+      </ComponentLink>
+    </WrapItem>
+  )
+
+  return (
+    <Wrap
+      className='component-links'
+      mt='2rem'
+      spacing='4'
+      overflow='unset'
+      {...rest}
+    >
+      {githubLink}
+      {themeComponentLink}
+      {npmLink}
+      {storybookLink}
+      {videoLink}
+    </Wrap>

```

Comment:
Could we replace this with a `ButtonGroup` and modify the `ComponentLink` to be a Button styled accordingly as Link ? I think that would clean up here this component a lot and be more chakra like

Reply:
Ahh, yes! I'm on it! 🚀

Reply:
@noobinthisgame I guess while I'm in here, do we still use the `ComponentLink`'s for Youtube and Storybook links? I've never seen them used in any of the docs.

Reply:
Let's keep them for now, but add a comment above that they are unused if that's really the case.